src/LogFileReader.cpp

```diff
@@ -75,11 +78,37 @@ bool LogFileReader::readAll(const QString& fileName)
 bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) const
 {
     // TODO implement this first
+    QStringList lines = line.split(',');
+    bool ok;
+    if(lines.count()!=3)
+        return false;
+    QStringList lines_1 = lines[0].split(':');
+    QStringList lines_2 = lines[1].split('.');
+    int value = lines_1[0].toInt(&ok);
+    if((!ok)||(value>24)||(value<0))
+        return false;
 
+    int m = lines_1[1].toInt(&ok);
```

Comment:
Just going to reiterate what Enoch said about variable names, make them nice and descriptive!